sdk/Dockerfile

```diff
@@ -4,7 +4,9 @@ RUN apt-get update && apt-get install -y \
   build-essential \
   libpq-dev
 RUN apt-get install -y python3-dev python3-pip
-RUN pip3 install psycopg2 boto3
-# protobuf tools for Python
-RUN pip3 install grpcio grpcio-tools
-WORKDIR /usr/src/approzium/sdk
+# tools for generating Python protobuf files
+RUN pip3 install grpcio-tools
+# install sdk dependencies
+COPY . /usr/src/approzium/sdk
+WORKDIR /usr/src/approzium/sdk/python
+RUN pip3 install -e .
```

Comment:
Instead of repeating dependencies for Approzium SDK, just move the source code over and install the package. The exception is grpcio-tools, which is only needed for development and not the user